server/services/lifecycle/lifecycle.js

```diff
@@ -72,10 +72,39 @@ module.exports = ({ strapi }) => {
               )
             })
         },
-        async afterUpdateMany() {
-          strapi.log.error(
-            `Meilisearch could not find an example on how to access the \`afterUpdateMany\` hook. Please consider making an issue to explain your use case`
-          )
+        async afterUpdateMany(event) {
+          const meilisearch = strapi
+            .plugin('meilisearch')
+            .service('meilisearch')
+
+          const nbrEntries = await contentTypeService.numberOfEntries({
+            contentType: contentTypeUid,
```

Comment:
Where this `contentTypeUid` come from?

Reply:
It comes from the function that subscribe a contentType to every hook line 14

```
async subscribeContentType({ contentType }) {
      const contentTypeUid = contentTypeService.getContentTypeUid({
        contentType: contentType,
      })
      ...
```